week-3/F-array-forEach/exercise.js

```diff
@@ -8,6 +8,7 @@
 */
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
+console.log(arr.map(x => { return (x % 3 === 0 && x % 5 === 0 && x > 0) ? "FizzBuzz" : (x % 3 === 0 && x > 0) ? "Fizz" : (x % 5 === 0 && x > 0) ? "Buzz" : x }));
```

Comment:
My feedback from last week's homework is the same here. Using multiple chained ternary expressions is usually a bad idea. It is hard to read. Please change this to an `if`/`else if`/`else` statement